benchmark_runner/common/clouds/IBM/ibm_operations.py

```diff
@@ -188,28 +182,70 @@ def ibm_disconnect(self):
         """
         self.__remote_ssh.disconnect()
 
+    def __get_latest_version(self):
+        """
+        This method return latest version
+        :return:
+        """
+        ocp_versions = OCPVersions()
+        if 'latest' in self.__install_ocp_version:
+            openshift_version_data = self.__install_ocp_version.split('-')
+            return ocp_versions.get_latest_version(latest_version=openshift_version_data[1])
+        else:
+            openshift_version_data = self.__install_ocp_version.split('.')
+            return ocp_versions.get_latest_version(latest_version=f'{openshift_version_data[0]}.{openshift_version_data[1]}')
```

Comment:
You're using named constants above but indices here.  Do it the same way in both places.

Reply:
Fixed, add constant LATEST_VERSION = 'latest'